docs/pages/pointfree.rst

```diff
@@ -285,8 +285,6 @@ API Reference
 
 .. autofunction:: returns.pointfree.bind_context_result
 
-.. autofunction:: returns.pointfree.bind_context_ioresult
```

Comment:
Why deleting it?

Reply:
Oh, it's because when I removed this line the `bind_context_ioresult` was not there (introduced by #590).
I have rebased my branch yesterday, I'll return this line!!